OpenSim/Simulation/Model/GeometryPath.cpp

```diff
@@ -81,7 +72,7 @@ GeometryPath::~GeometryPath()
 void GeometryPath::setNull()
 {
     setAuthors("Peter Loan");
-    _maSolver = NULL;
+    _maSolver.reset();
```

Comment:
Alternatively, I will get rid of `setNull()` and do everything in the constructor.